neard/src/cli.rs

```diff
@@ -132,6 +137,10 @@ pub(super) enum NeardSubCommand {
     /// View DB state.
     #[clap(name = "view_state")]
     StateViewer(StateViewerSubCommand),
+    /// Recompresses the entire storage.  This is a slow operation which reads
+    /// all the data from the database and writes them down to a new copy of the
+    /// database.
+    RecompressStorage(RecompressStorageSubCommand),
```

Comment:
Why does it not have `#[clap(name = ...)]` :)

Reply:
It’s not necessary. Clap is smart enough to convert it to `recompress-storage`.